README.md

```diff
@@ -53,17 +53,12 @@ If you plan on extending the features of MSync 2, please copy and use the sample
 
 You need to install some things manually on the Server in order for MSync to work. 
 
-First you need libmysql on windows or libmysqlclient on linux. You can find them here:
-Windows: https://github.com/FredyH/MySQLOO/raw/master/MySQL/lib/windows/libmysql.dll
-Linux: https://github.com/FredyH/MySQLOO/raw/master/MySQL/lib/linux/libmysqlclient.so.18
-Put the file in the same folder where your scrds.exe or your scrds_linux file is.
-
-Then you need MySQLoo 9 or higher. You can download MySQLoo from here:
-https://github.com/FredyH/MySQLOO/releases
-Select win32 if your server runs on windows, otherwise use the linux dll.
-Put the dll file in your lua/bin folder. If you don't have one, create it.
-it should look like this:
+First you need gmsv_mysqloo_win32.dll on windows or gmsv_mysqloo_linux.dll on linux. You can find them here:
```

Comment:
There is also a linux32 and linux64 binary. I'd also mention line 71 up here directly. But with a more generic message. Something like
```
First you need to find out if your Server is running on Windows or on Linux, and if your server is 32bit or 64bit. The easiest way to find that out is by using the following command:
``lua_run print("Operating System: ", jit.os) print("Server Arch: ", jit.arch)``

As for the server arch: x86 means 32bit while x64 means 64bit

Example output:
\`\`\`
Operating System:       Linux
Server Arch:    x86
\`\`\`
So in our examples case we need the ``gmsv_mysqloo_linux32.dll``

You can then find the appropriate file here: https://github.com/FredyH/MySQLOO/releases
```